content/docs/300-developers/100-typescript.mdx

```diff
@@ -32,7 +32,7 @@ const clientOptions = {
 const kineticClient = await KineticSdk.setup(clientOptions)
 ```
 
-Don't have an App Index? Register your app on our Developer Portal so you can get your App Index that allows you to transact with our SDKs and earn via the KRE.
+Don't have an App Index? Register your app on our [Developer Portal](https://portal.kin.org/) so you can get your App Index that allows you to transact with our SDKs and earn via the KRE.
```

Comment:
Hi. there's a button straight below this with the link to how to register on the portal.
![image](https://user-images.githubusercontent.com/62516140/204398098-0859f0d8-e4e5-45ca-8f16-245e825e5b25.png)